web/src/ApiButton.test.tsx

```diff
@@ -335,6 +333,59 @@ describe("ApiButton", () => {
 
     expect(error).toEqual("Error submitting button click: broken!")
   })
+
+  it("when requiresConfirmation is set, a second confirmation click to submit", async () => {
+    const b = makeUIButton()
+    b.spec!.requiresConfirmation = true
```

Comment:
(small nit) it might be nice to have `requiresConfirmation` as a param that gets passed into this function! my not-so-secret "secret" agenda is to move the test data functions into being more configurable, so we don't have to do manual manipulation and tweaking in tests.

Reply:
yes, thanks!